src/pages/Staking/index.tsx

```diff
@@ -505,9 +506,7 @@ const Staking = (): JSX.Element => {
   };
   const availableLockTime = getAvailableLockTime();
 
-  const renderAvailableBalanceLabel = () => {
-    return availableBalance === undefined ? '-' : displayMonetaryAmount(availableBalance);
-  };
+  const availableMonetaryBalance = displayMonetaryAmount(availableBalance);
```

Comment:
Do we not need to handle undefined anymore? Rendering an en dash when a balance is unavailable is something that's done in quite a few places so would be nice to keep it consistent.

Reply:
I still use the dash, I just don't use this function any more.

https://github.com/interlay/interbtc-ui/pull/388/files/efa705d1cad02008c8a6d830e8acd1e3860b272f#diff-0c884daa41a373980d483b5e6d61c028b63e2d2176290e4ad7d9313bfb3a0074R750

Reply:
Ok sounds good :) long as we don't have any regression then that's fine by me